configs/workbench.json

```diff
@@ -1,11 +1,38 @@
 {
   "index_name": "workbench",
-  "start_urls": ["https://workbench.gusto.com/"],
+  "start_urls": [
+    {
+      "url": "https://workbench.gusto.com/"
+    },
+    {
+      "url": "https://workbench.gusto.com/getting-started/",
+      "page_rank": 1
+    },
+    {
+      "url": "https://workbench.gusto.com/foundations/",
+      "page_rank": 2
+    },
+    {
+      "url": "https://workbench.gusto.com/components/",
+      "page_rank": 3
+    },
+    {
+      "url": "https://workbench.gusto.com/patterns/"
+    },
+    {
+      "url": "https://workbench.gusto.com/content/"
+    },
+    {
+      "url": "https://workbench.gusto.com/resources/"
+    },
+    {
+      "url": "https://workbench.gusto.com/updates/"
+    }
+  ],
   "stop_urls": ["\\/\\?"],
-  "sitemap_urls": ["https://workbench.gusto.com/sitemap/sitemap-0.xml"],
```

Comment:
Specifying the sitemap is always a good point, we do not chose one or the other but it complements URLs found from start_urls, I'd recommend to keep it

Reply:
Thank you for pointing that out. I added it back. 👍